src/main/resources/META-INF/mods.toml

```diff
@@ -1,18 +1,17 @@
 modLoader="javafml"
 loaderVersion="[24,]"
-issueTrackerURL="http://www.minecraftforge.net/"
+issueTrackerURL="https://minecraftforge.net/"
```

Comment:
Shouldn't this link be changed to "https://forums.minecraftforge.net/forum/18-support-bug-reports/" or "https://github.com/MinecraftForge/MinecraftForge/issues", or "issueTrackerURL" to "displayURL"?